iOS Client/ArticlesController/MBArticlesViewController.swift

```diff
@@ -85,17 +95,26 @@ class MBArticlesViewController: UIViewController, UITableViewDelegate, UITableVi
     func newState(state: MBAppState) {
         switch state.articleState.articles {
         case .initial:
-            break //Do nothing if articles haven't tried to load
+            break
         case .loading:
-            break //Do something here to indicate loading
+            if case Loaded<[MBArticle]>.loading = self.currentState.articles {
```

Comment:
could just be `if case .loading = self.currentState.articles` I think.

Reply:
changed it